src/formats/tinkerformat.cpp

```diff
@@ -152,8 +153,6 @@ namespace OpenBabel
     bool mmffTypes = pConv->IsOption("m",OBConversion::OUTOPTIONS) != NULL;
     bool mm3Types = pConv->IsOption("3",OBConversion::OUTOPTIONS) != NULL;
     bool classTypes = pConv->IsOption("c", OBConversion::OUTOPTIONS) != NULL;
-    if (pmol->GetData("Atom Class") == NULL)
-      classTypes = false;
```

Comment:
Hmm. The point of this test was to switch to a different atom type (MM2) if atom classes aren't available. Not quite sure how to do that easily now (except looping through the atoms).

Reply:
Not that this should hold up the change, though.

Reply:
With the caveat that I'm not familiar with the format, I can see how the behavior is different, but I  prefer this behavior. If you leave out atom classes but specify 'c', you want to know that there's something not right. I could add a warning, but I'm not keen on an automatic fallback.